### ISSUE TITLE
Single character unquoted strings cause mismatched input error

### ISSUE THREAD

**Issue by dnicolson** [Serchinastico/Kin#52]
Consider the following code excerpt:
```
B3AF46E021071CA6002211EE /* C */ = {
    isa = PBXGroup;
    children = (
    );
    name = C;
    sourceTree = "<group>";
};
```
Because the path is only a single character it causes the following error:

`mismatched input 'C' expecting {QUOTED_STRING, NON_QUOTED_STRING}`

The problem does not occur if the string is quoted.

**Comment by Serchinastico** [comment]
I apologise for the delay here @dnicolson 🙏🏼 , I don't know how this went under the radar for so long. I'll find some time to address the issue in the upcoming days/weeks

**Comment by Serchinastico** [comment]
Should be fixed in [2.1.2](https://github.com/Serchinastico/Kin/releases/tag/2.1.2)! 🎉
